### PR TITLE
Fix some issues with the CI

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: Jellyfin.Data/Jellyfin.Data.csproj [hunk 1]
```diff
@@ -4,7 +4,6 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TreatWarningsAsErrors Condition=" '$(Configuration)' == 'Release' ">true</TreatWarningsAsErrors>
```
Comment (Bond-009) [inline Jellyfin.Data/Jellyfin.Data.csproj:7]: Why did you remove this?

Comment (dkanada) [reply Jellyfin.Data/Jellyfin.Data.csproj:7]: It caused a ton of errors that were causing build failures.

Comment (crobibero) [reply Jellyfin.Data/Jellyfin.Data.csproj:7]: looks like all doc errors. Is there a way to disable just that warning?

Comment (barronpm) [reply Jellyfin.Data/Jellyfin.Data.csproj:7]: You can in the csproj file by adding `<NoWarn>1591</NoWarn> in the `PropertyGroup` tag

File: Jellyfin.Data/Jellyfin.Data.csproj [hunk 2]
```diff
@@ -4,7 +4,6 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
```
Comment (crobibero) [inline Jellyfin.Data/Jellyfin.Data.csproj:6]:
```suggestion
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <TreatWarningsAsErrors Condition=" '$(Configuration)' == 'Release' ">true</TreatWarningsAsErrors>
    <NoWarn Condition=" '$(Configuration)' == 'Release' ">1591,1573</NoWarn>
```

Comment (barronpm) [reply Jellyfin.Data/Jellyfin.Data.csproj:6]: entries in NoWarn are semicolon delimited, afaik

Comment (crobibero) [reply Jellyfin.Data/Jellyfin.Data.csproj:6]: Just tested, both `;` and `,` work so 🤷

Comment (Bond-009) [reply Jellyfin.Data/Jellyfin.Data.csproj:6]: This misses the point of all the pragma warnings, that new files need to be fully documented.

File: Jellyfin.Data/Jellyfin.Data.csproj (hunk 1 above)
Comment (dkanada) [reply Jellyfin.Data/Jellyfin.Data.csproj:7]: I think we should merge this to fix the CI and then debate about the ideal solution in chat to avoid throwing any more errors.
